packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/notifications/RealmNotificationsTests.kt

```diff
@@ -179,7 +179,6 @@ class RealmNotificationsTests : FlowableTests {
                 assertEquals(VersionId(startingVersion.version + 2), realmChange.realm.version())
             }
 
-            realm.write { /* Do nothing */ }
```

Comment:
unecessary?

Reply:
No, it will trigger a notification on observer1, so we can verify that observer2 was correctly canceled.